libraries/AP_HAL_ChibiOS/hwdef/luminousbee5/hwdef.dat

```diff
@@ -202,16 +223,22 @@ define HAL_GPIO_C_LED_PIN 2
 # battery setup
 define HAL_BATT_VOLT_PIN 14
 define HAL_BATT_CURR_PIN 15
-define HAL_BATT_VOLT_SCALE 10.1
-define HAL_BATT_CURR_SCALE 17.0
+define HAL_BATT_VOLT_SCALE 3.28
+define HAL_BATT_CURR_SCALE 3.06
 
 # setup serial port defaults for ESP8266
 define HAL_SERIAL5_PROTOCOL SerialProtocol_MAVLink
 define HAL_SERIAL5_BAUD 921600
 
+# 6 PWM available by default
+define BOARD_PWM_COUNT_DEFAULT 7
+
+
 # two IMUs
 #IMU Invensense SPI:icm20608 ROTATION_YAW_180
-IMU BMI088       SPI:bmi088_a SPI:bmi088_g ROTATION_ROLL_180_YAW_90
```

Comment:
this will break things for any user using an older board with a BMI088. Is that really what you want to do? 
I think we should make it work with both older and newer boards

Reply:
In this case as I have already written , in my last comment . The new boards will not use this IMU . So keeping it commented out should not create future problems .